src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
```

Comment:
We should use descriptive property names - I am guessing that `p` property is person, but better will be:
```suggestion
    ? people.filter((person) => person.sex === 'm' && Math.ceil(person.died / 100) === century)
```

Reply:
We should use descriptive property names - I am guessing that `m` means `male`. If you have control over data object it should be:
 ```suggestion
     ? people.filter(person => person.sex === 'male' && Math.ceil(person.died / 100) === century)
```